### DISCUSSION TITLE
Genetic Apex and Promo packs from new game

### DISCUSSION THREAD

**Discussion by anilkrl** [tcgdex/cards-database#581] [Help & Support] [answered]
Hello, first of all, thank you very much for providing this API. When will the Genetic Apex and Promo packs from the Pokémon TCG Pocket game be fully added with their images?

**Comment by Aviortheking** [comment, marked as answer]
Hi Genetic Apex now have images !

I am adding Promos currently and it will be available by the end of the week !

**Reply by liamvgallagher** [reply]
Hello, thank you so much for your contributions to the project! I am curious about how you go about sourcing the images for the PTCGP cards. I would be happy to help as packs come out in the future as well. Thanks.

**Reply by Aviortheking** [reply]
From different websites that display them mostly.
Help is always appreciated.

I am still trying to engineer over the real app to get more infos but that's quite hard for me

**Comment by liamvgallagher** [comment]
I unzipped the APK, but I remember the app downloading additional assets
when I first downloaded. The card assets are likely downloaded then.
Intercepting the network traffic when that happens could lead to something.
I will look into it.

**Reply by Aviortheking** [reply]
Yep thats the main thing, the app download oaver https and certificates are hard to get around
